analysis/paladinholy/src/modules/items/UntemperedDedication.tsx

```diff
@@ -0,0 +1,111 @@
+import SPELLS from 'common/SPELLS';
+import conduits from 'common/SPELLS/shadowlands/conduits';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import { ApplyBuffStackEvent, ApplyBuffEvent, RemoveBuffEvent } from 'parser/core/Events';
+import SUGGESTION_IMPORTANCE from 'parser/core/ISSUE_IMPORTANCE';
+import { When, ThresholdStyle } from 'parser/core/ParseResults';
+
+class UntemperedDedication extends Analyzer {
+  // Warcraft Logs seems to have issues with the stack status at fight start
+  // we do not know how many stacks the character has and how long it has until expiration
+  // in WoWA we have a prepull apply buff with the same lack of info
+  // Realistically, people start at 5 stacks, or _isFullStack is going to turn to false on the first applyBuff or applyBuffStack
+  // the only issue is that the first LOTM may be considered wasteful when it is not.
+  _isFullStack = true;
+  // There is also no buff event generated when LOTM is cast while there are already 5 stacks
+  // so we are maintaining the date of the last refresh here.
+  _lastFullStackRefresh = 0;
+  // a counter for when LOTM stacks are dropped
+  _totalDrops = 0;
+  // tracker for number of current stacks
+  _currentStacks = 5;
```

Comment:
I wouldn't assume you start at 5.
The vast majority of players are not going to start at five

If you are really concerned about the edge case I'd capture the first n LoTM until we see a buff applied/buff stack gained/buff refreshed/buff removed event and then deal with them